build_wheel.sh

```diff
@@ -366,7 +366,14 @@ if [[ -z "$EBROOTGENTOO" ]]; then
 	module --force purge
 	module load nixpkgs gcc/7.3.0
 else
-	module --force purge
+	if [[ -z "$EBVERSIONARCH" ]]; then
+		module --force purge
+		module load arch/sse3
+	else
+		ARCH_TO_LOAD="$EBVERSIONARCH"
+		module --force purge
+		module load arch/$ARCH_TO_LOAD
```

Comment:
Why defining ARCH_TO_LOAD if it is not reused? EBVERSIONARCH would do :)

Reply:
It is reused, 2 lines later.